specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/stable/2021-11-12/hanaonazure.json

```diff
@@ -0,0 +1,707 @@
+        },
+        "isDataAction": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether the operation applies to data-plane."
+        }
+      },
+      "description": "HANA operation information"
+    },
+    "Display": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource provider name. This form is also expected to include the publisher/company responsible. Use Title Casing. Begin with \"Microsoft\" for 1st party services."
+        },
+        "resource": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource type related to this action/operation. This form should match the public documentation for the resource provider. Use Title Casing. For examples, refer to the “name” section."
+        },
+        "operation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly name for the operation as shown to the user. This name should be concise (to fit in drop downs), but clear (self-documenting). Use Title Casing and include the entity/resource to which it applies."
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly description for the operation as shown to the user. This description should be thorough, yet concise. It will be used in tool-tips and detailed views."
+        },
+        "origin": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The intended executor of the operation; governs the display of the operation in the RBAC UX and the audit logs UX. Default value is 'user,system'"
+        }
+      },
+      "description": "Detailed HANA operation information"
+    },
+    "ErrorResponse": {
+      "type": "object",
+      "description": "Describes the format of Error response.",
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "Describes the error object.",
+          "properties": {
+            "code": {
+              "readOnly": true,
+              "description": "Error code",
+              "type": "string"
+            },
+            "message": {
+              "readOnly": true,
+              "description": "Error message indicating why the operation failed.",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "Tags": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags field of the HANA instance."
+        }
+      },
+      "description": "Tags field of the HANA instance."
```

Comment:
The diff b/w this file and json file for preview version shows some unexpected deletions from this line forward related to monitoring details, why is that so?

Reply:
some of the sap monitor related definitions are not required as we are not supporting the monitor commands.

Reply:
added the apis